src/libprojectM/Renderer/Renderer.cpp

```diff
@@ -229,6 +236,16 @@ void Renderer::ResetTextures()
 void Renderer::SetupPass1(const Pipeline& pipeline, const PipelineContext& pipelineContext)
 {
 	totalframes++;
+	this->currentTime = duration_cast<milliseconds>(system_clock::now().time_since_epoch());
```

Comment:
perhaps we could avoid doing this work if we're not outputting FPS?

Reply:
You are right. If FPS is not being displayed there is no need to check this timer. I will make that change.